flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -635,8 +644,18 @@
 	 * The default number of archived jobs for the jobmanager
 	 */
 	public static final int DEFAULT_JOB_MANAGER_WEB_ARCHIVE_COUNT = 5;
-	
-	
+
+	/**
+	 * By default, submitting jobs from the web-frontend is allowed.
+	 */
+	public static final boolean DEFAULT_JOB_MANAGER_WEB_SUBMISSION = true;
+
+	/**
+	 * Default directory for uploaded file storage for the Web frontend.
+	 */
+	public static final String DEFAULT_JOB_MANAGER_WEB_UPLOAD_DIR =
+			(System.getProperty("java.io.tmpdir") == null ? "/tmp" : System.getProperty("java.io.tmpdir")) + "/webmonitor/";
```

Comment:
No. In case user wants them persisted, they should specify a directory themselves. Otherwise, as long as the OS isn't rebooted, the jars will stay there.

Also, perhaps a better idea will be to not have a default directory, and determine it while initializing the Web Monitor. That way, we can get the directory on the job manager machine, instead of the machine where the jar was built.